test/IRGen/objc_implementation.m

```diff
@@ -0,0 +1,26 @@
+// REQUIRES: disabled
```

Comment:
To clarify: This test intentionally doesn’t pass, but it also doesn’t run normally and arguably shouldn’t be checked in at all. It’s just a convenient way to see the IR that clang and swift both generate for equivalent implementations.